README.md

```diff
@@ -79,19 +79,26 @@ layout.addView(name)
 layout.addView(button)
 ```
 
-A DSL makes the same logic easy to read, easy to write and there're no runtime overhead. Just try it!
+A DSL makes the same logic easy to read, easy to write and there is no runtime overhead. Here it is again:
+
+```kotlin
+verticalLayout {
+  val name = editText()
+  button("Say Hello") {
+    onClick { toast("Hello, ${name.text}!") }
+  }
+}
+```
 
 ### Why not Scaloid?
```

Comment:
Personally I think Why not Scaloid is fine since it's basically focusing on why this if there's Scaloid.